### PR TITLE
Tasks 4, 5 and 6

### PR DESCRIPTION
**Pull request by TulasiJagan49**
@praveengarimella Please review the added code in Register Branch.

### CODE REVIEW

File: project1/templates/register.html [hunk 1]
```diff
@@ -18,6 +18,12 @@ <h1> Welcome {{name}} </h1>
                     <label class="form-check-label" for="exampleCheck1">Remember me</label>
         </div>
   			<button type="submit" class="btn btn-primary">Submit</button>
+
+  			<!-- <div class="form-group form-check">
+                    <input type="checkbox" class="form-check-input" id="exampleCheck1">
+                    <label class="form-check-label" for="exampleCheck1">Check me out</label>
+            </div> -->
```
Comment (praveengarimella) [inline project1/templates/register.html]: Clean this up

File: project1/application.py [hunk 2]
```diff
@@ -14,24 +18,60 @@
...
+        self.password = bcrypt.encrypt(password)
+        self.timestamp = datetime.datetime.now()
 
 
 @app.route("/")
 def index():
-    return render_template("index.html")
+    # if 'name' in session:
+    #     name = session["name"]
+    #     return render_template("index.html", flag = True, name = name)
```
Comment (praveengarimella) [inline project1/application.py]: Clean this up

File: project1/application.py [hunk 3]
```diff
@@ -14,24 +18,60 @@
...
-		name = request.form.get("name")
-		password = request.form.get("password")
-		print(f"Printing name recieved from User: {name}")
-		return render_template("register.html", flag = flag, name = name)
-	return render_template("register.html", flag = flag)
+    # if session.get("remember") is None:
+    #     session["name"] = ""
+    #     session["password"] = ""
+    # elif session.get("remember") == "yes": // Will uncomment it when logout is implemented.
+    #     return render_template("register.html", flag = True, name = session["name"])
```
Comment (praveengarimella) [inline project1/application.py]: Clean this  up

File: project1/application.py [hunk 4]
```diff
@@ -14,24 +18,60 @@
...
+    #     session["name"] = ""
+    #     session["password"] = ""
+    # elif session.get("remember") == "yes": // Will uncomment it when logout is implemented.
+    #     return render_template("register.html", flag = True, name = session["name"])   
+    if request.method == "POST":
+        # session["name"] = (request.form.get("name"))
+        # session["password"] = (request.form.get("password"))
+        # session["remember"] = (request.form.get("remember"))
+        # print("Printing name recieved from User:" + session["name"]) // This 
+        # is for Task-3 purpose so commented it out.
```
Comment (praveengarimella) [inline project1/application.py]: Clean this up

File: project1/application.py [hunk 5]
```diff
@@ -14,24 +18,60 @@
...
+        db.session.add(User(request.form.get("name"), request.form.get("password")))
+        # db.execute("INSERT INTO users (user, password,timestamp) VALUES (:username, :password, :timestamp)",
+        #     {"username":request.form.get("user"), "password":request.form.get("password"), "timestamp":datetime.datetime.now()})
```
Comment (praveengarimella) [inline project1/application.py]: Clean this up

File: project1/templates/admin.html [hunk 6]
```diff
@@ -0,0 +1,25 @@
+{% extends "layout.html" %}
```
Comment (praveengarimella) [inline project1/templates/admin.html:1]: Nice 👍

**Review by praveengarimella — APPROVED** [review]
Nice work  👍 

You should avoid the bin files in the repo.

File: project1/templates/register.html (hunk 1 above)
Comment (TulasiJagan49) [reply project1/templates/register.html]: Cleaned it up.

File: project1/application.py (hunk 2 above)
Comment (TulasiJagan49) [reply project1/application.py]: Removed.

File: project1/application.py (hunk 4 above)
Comment (TulasiJagan49) [reply project1/application.py]: Removed.

File: project1/application.py (hunk 5 above)
Comment (TulasiJagan49) [reply project1/application.py]: Removed.

**Review by praveengarimella — COMMENTED** [review]
👍

File: project1/templates/admin.html (hunk 6 above)
Comment (TulasiJagan49) [reply project1/templates/admin.html:1]: Thanks

**Pull request merged**